modules/github.py

```diff
@@ -98,21 +98,29 @@ async def format_highlight_block(self, url: str, line_adjustment: int = 10) -> d
         _min_boundary = highlighted_line - 1 - bound_adj
         _max_boundary = highlighted_line - 1 + bound_adj
 
-        # loop through all the lines, and adjust the formatting
-        msg = "```ansi\n"
+        # get the file extension to format nicely
+        file = match["file"]
+        extension = file.split(".")[1]
```

Comment:
Ideally I would have done `rsplit(".")[-1]` as this allows for filenames that include a period:-

```py
In [7]: "foo.bar.py".rsplit(".")[-1]
Out[7]: 'py'

In [8]: "foo.bar.py".split(".")[1]
Out[8]: 'bar'
```